revocation/result/results.go

```diff
@@ -116,6 +120,6 @@ type CertRevocationResult struct {
 
 	// RevocationMethod is the method used to check the revocation status of the
 	// certificate, including Unknown(0), MethodOCSP(1), MethodCRL(2) and
-	// OCSPFallbackCRL(3)
-	RevocationMethod int
+	// MethodOCSPFallbackCRL(3)
+	RevocationMethod revocation.Method
```

Comment:
```suggestion
	RevocationMethod Method
```

Reply:
However, we cannot call `result.go` is in `result` package and cannot call `Method`.

Reply:
Refactored and moved the `revocation.Method` to `result.RevocationMethod`